docs/gc-font-tool.cpp

```diff
@@ -123,6 +135,22 @@ const uint16_t windows_1252_font_table[] =
 	SEQUENCE_32(0xA0),
 	SEQUENCE_64(0xC0),
 };
+static_assert(std::size(windows_1252_font_table) == 224);
+
+// List of horizontal glyph widths
+const uint8_t windows_1252_widths_table[] =
```

Comment:
Just curious, how did you get these arrays?

Should you use a `std::array` instead?

Reply:
It's not clear to me whether something as trivial as glyph widths falls under copyright, but I'd rather be safe than sorry, so I've closed this PR.